test/visual/stroke_projection.js

```diff
@@ -1,307 +0,0 @@
-                percentage: 0.001,
-                width: 600,
-                height: 900,
-                fabricClass: 'Canvas',
-                disabled: fabric.isLikelyNode,
-              });
-            });
-          });
-        });
-      });
-    });
-  }
-
-  for (let [caseName, casePoints] of Object.entries(selectedMiterLimitCases)) {
-    const builder = fabric.Polygon;
-    [5, 20, 120].forEach((strokeMiterLimit) => {
-      [true, false].forEach((strokeUniform) => {
-        tests.push({
-          test: `${caseName} with strokeMiterLimit=${strokeMiterLimit}, strokeUniform=${strokeUniform} values`,
-          code: function (canvas, callback) {
-            renderStrokeTest(
-              canvas,
-              {
-                builder,
-                points: casePoints,
-              },
-              {
-                strokeLineJoin: 'miter',
-                strokeUniform,
-                strokeMiterLimit,
-              }
-            );
-            callback(canvas.lowerCanvasEl);
-          },
-          golden: `stroke-projection/strokeLineJoin/miter-limit/${caseName}-${
-            strokeUniform ? 'uniform-' : ''
-          }-${strokeMiterLimit}miterLimit.png`,
-          percentage: 0.001,
-          width: 600,
-          height: 900,
-          fabricClass: 'Canvas',
-          disabled: fabric.isLikelyNode,
-        });
-      });
-    });
-  }
-
-  [fabric.Polyline, fabric.Polygon].forEach((builder) => {
-    const builderType = builder.prototype.type,
-      isPolygon = builderType === 'polygon';
-    ['square', 'round'].forEach((strokeLineCap) => {
-      [true, false].forEach((strokeUniform) => {
-        [
-          [0, 0],
-          [0, 30],
-          [20, 0],
-          [25, 35],
-        ].forEach(([skewX, skewY]) => {
-          tests.push({
-            test: `${testSinglePoint.name} of type ${builderType} with strokeLineCap=${strokeLineCap}, strokeUniform=${strokeUniform}, skewX=${skewX}, skewY=${skewY}`,
-            code: function (canvas, callback) {
-              renderStrokeTest(
-                canvas,
-                {
-                  builder,
-                  points: testSinglePoint.points,
-                },
-                {
-                  strokeLineCap,
-                  strokeUniform,
-                  skewX,
-                  skewY,
-                }
-              );
-              callback(canvas.lowerCanvasEl);
-            },
-            golden: `stroke-projection/${
-              isPolygon ? 'strokeLineJoin' : 'strokeLineCap'
-            }/${testSinglePoint.name}-${strokeLineCap}${
-              strokeUniform ? '-uniform-' : ''
-            }-${skewX}skewX-${skewY}skewY.png`,
-            percentage: 0.001,
-            width: 600,
-            height: 900,
-            fabricClass: 'Canvas',
-            disabled: fabric.isLikelyNode,
-          });
-        });
-      });
-    });
-  });
-
-  tests.forEach(visualTestLoop(QUnit));
-});
```

Comment:
we shouldn't loose those. Have these been ported over?

Reply:
i don't want to delete it because i will forget it exists, it may be in git yes, but if i don't remember is like is not there.
just leave the file alive in the folder, is enough.